.github/workflows/lmi-no-code.yml

```diff
@@ -186,54 +188,54 @@ jobs:
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=s3://djl-llm/mistral-7b/" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code mistral-7b
           docker rm -f $(docker ps -aq)
       - name: GPTNeox lmi container
-        if: ${{ matrix.container }} == "deepspeed"
         working-directory: tests/integration
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=s3://djl-llm/gpt-neox-20b/" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code gpt-neox
           docker rm -f $(docker ps -aq)
       - name: Phi2 lmi container
-        if: ${{ matrix.container }} == "deepspeed"
         working-directory: tests/integration
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=microsoft/phi-2" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code phi-2
           docker rm -f $(docker ps -aq)
       - name: Baichun lmi container
         working-directory: tests/integration
         run: |
           rm -rf models
-          echo -en "HF_MODEL_ID=s3://djl-llm/baichuan2-13b/\nHF_TRUST_REMOTE_CODE=true" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          echo -en "HF_MODEL_ID=s3://djl-llm/baichuan2-13b/\nHF_MODEL_TRUST_REMOTE_CODE=true" > docker_env
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code baichuan-13b
           docker rm -f $(docker ps -aq)
       - name: Qwen-1.5 lmi container
         working-directory: tests/integration
+        if: ${{ matrix.container == 'deepspeed' }}
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=Qwen/Qwen1.5-14B" > docker_env
```

Comment:
qwen 1.5 is qwen2?

Reply:
yea https://huggingface.co/Qwen/Qwen1.5-0.5B/blob/main/config.json